.github/workflows/cpu-ci.yml

```diff
@@ -14,11 +14,13 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
RAPIDS has dropped support for 3.7, so we have too. We're starting to make active use of Python 3.8 features like `typing.Protocol`, so might want to omit 3.7 here.

Reply:
done